src/App.tsx

```diff
@@ -1,7 +1,41 @@
-import React from 'react';
+import { Component, ReactNode } from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+type State = {
+  pressedKey : string;
+};
+
+export class App extends Component<{}, State> {
+  state: Readonly<State> = {
+    pressedKey: '',
+  };
+
+  text = 'Nothing was pressed yet';
+
+  componentDidMount() {
+    document.addEventListener('keyup', this.isClicked);
+  }
+
+  componentWillUnmount() {
+    document.removeEventListener('keyup', this.isClicked);
+  }
+
+  isClicked = (event: KeyboardEvent) => {
+    this.setState({ pressedKey: `${event.key}` });
+  };
+
+  render(): ReactNode {
+    const button = this.state.pressedKey;
```

Comment:
```suggestion
    const { pressedKey } = this.state;
```
You can just use pressedKey from state